### PR TITLE
fix(android): changed from SegmentedControlIOS to SegmentedControlTab

### PR DESCRIPTION
**Pull request by sfeu**
Changed to SegmentedControlTab (MIT License), which also works with Android

### CODE REVIEW

File: src/pages/Contact/ContactInfos/ContactInfos.tsx [hunk 1]
```diff
@@ -30,8 +30,8 @@ export class ContactInfos extends PureComponent<
   public state = {
     index: 2,
   };
-  public setSegmentedControlState = event =>
-    this.setState({index: event.nativeEvent.selectedSegmentIndex});
+  public setSegmentedControlState = index =>
+    this.setState({index: index});
```
Comment (Mokto) [inline src/pages/Contact/ContactInfos/ContactInfos.tsx]: You could use object literal shorthand. Have you installed tslint in your IDE ?

File: src/pages/Contact/ContactInfos/ContactInfos.tsx [hunk 2]
```diff
@@ -41,12 +41,13 @@ export class ContactInfos extends PureComponent<
...
     return [
-      <SegmentedControlIOS
+      <SegmentedControlTab
         key={0}
-        tintColor={primaryColor}
+ //       tintColor={primaryColor}
         values={values}
         selectedIndex={index}
-        onChange={this.setSegmentedControlState}
+//        onChange={this.setSegmentedControlState}
```
Comment (Mokto) [inline src/pages/Contact/ContactInfos/ContactInfos.tsx]: You could remove this comment.

File: src/pages/Contact/ContactInfos/ContactInfos.tsx [hunk 3]
```diff
@@ -41,12 +41,13 @@ export class ContactInfos extends PureComponent<
       index === 0 ? Family : index === 1 ? Meet : index === 2 ? Work : Contact;
 
     return [
-      <SegmentedControlIOS
+      <SegmentedControlTab
         key={0}
-        tintColor={primaryColor}
+ //       tintColor={primaryColor}
```
Comment (Mokto) [inline src/pages/Contact/ContactInfos/ContactInfos.tsx]: We need to use the right color for the segmented control tab ;)

**Review by Mokto — CHANGES_REQUESTED** [review]

**Comment by Mokto** [comment]
Hi @sfeu,

Thanks ! You are officially the first side contributor to chandler ! It makes us really happy ;)

I made some comments on your code.

**Comment by sfeu** [comment]
Hi @Mokto,

thanks for your support! I tried to implement the requested changes.

**Comment by Mokto** [comment]
Thanks @sfeu !

**Pull request merged**